wasm/typescript/test/browser/demo-app/read-parameter-files-load-sample-inputs.ts

```diff
@@ -1,30 +1,43 @@
-export default async function readParameterFilesLoadSampleInputs (model, preRun=false) {
-  const dataUrl = 'https://w3s.link/ipfs/bafybeifq7m3mb4m3mgbhft2vqejjgwxx4azmmzuxwf5cubajpyaf4hq2gq/data/input'
+export default async function readParameterFilesLoadSampleInputs(
+  model,
+  preRun = false
+) {
+  const dataUrl =
+    "https://itk.mypinata.cloud/ipfs/bafybeifq7m3mb4m3mgbhft2vqejjgwxx4azmmzuxwf5cubajpyaf4hq2gq/data/input";
```

Comment:
Thanks Matt. Do I understand correctly that the replacement of "w3s.link" with "itk.mypinata.cloud" is the only fundamental change here?

Did you use a tool to adjust the reformatting and add semi-colons?  If so, can you tell me how you did it?

Reply:
Thanks! For the next time, I think it would be nicer to have the reformatting in a separate commit, but for now, it's fine to me 👍